packages/sakuli-legacy/src/context/common/actions/clipboard.function.ts

```diff
@@ -12,10 +12,12 @@ export const ClipboardApi = {
         await clipboard.copy(text);
     },
     async pasteClipboard() {
```

Comment:
Shouldn't i be something like

```typescript
for(let key of pasteShortcut()) {
  await keyboard.pressKey(k);
  await keyboard.releaseKey(k); 
})
```

Reply:
keyboard.type(...) handles combinations in a way that both keys are pressed simultaneously.
Running it via loop would give a wrong series of keyevents, since it would be

key1->down
key1->up
key2->down
key2->up

Reply:
ahhh i see this is regarding the CTRL+C / CTRL+V keys.